omod/src/main/resources/apps/coreappsAppTemplates.json

```diff
@@ -94,6 +94,18 @@
                 "defaultValue": ""
             }
         ]
+    },
+    {
+        "id": "coreapps.template.baseTemplate",
```

Comment:
@rkorytkowski What do you think about this ? We can't set in the template fields like icon or label like we could in app description and this is the cleanest way  I've found without modifying  [this](https://github.com/openmrs/openmrs-module-appframework/blob/master/api/src/main/java/org/openmrs/module/appframework/domain/AppTemplate.java)